packages/wallets/src/components/WalletsCarousel/WalletsCarousel.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { useEffect, useRef, useState } from 'react';
 import { useActiveWalletAccount } from '@deriv/api-v2';
 import { AccountsList } from '../AccountsList';
-import { WalletText } from '../Base';
 import { WalletsCarouselContent } from '../WalletsCarouselContent';
 import { WalletsCarouselHeader } from '../WalletsCarouselHeader';
 import './WalletsCarousel.scss';
 
 const WalletsCarousel: React.FC = () => {
     const [isWalletSettled, setIsWalletSettled] = useState(true);
-    const [hideWalletsCarouselHeader, setHideWalletsCarouselHeader] = useState(false);
+    const [hideWalletsCarouselHeader, setHideWalletsCarouselHeader] = useState(true);
```

Comment:
Changed default value to true to solve issue where carousel header flicker on initial load

https://github.com/binary-com/deriv-app/assets/142860499/a837b33a-0acf-4485-b2c0-4c22d6e8923b